cmake/Modules/FindSYCLToolkit.cmake

```diff
@@ -0,0 +1,70 @@
+# This will define the following variables:
+# SYCL_FOUND               : True if the system has the SYCL library.
+# SYCL_INCLUDE_DIR         : Include directories needed to use SYCL.
+# SYCL_LIBRARY_DIR         ：The path to the SYCL library.
+# SYCL_LIBRARY             : SYCL library fullname.
+
+include(FindPackageHandleStandardArgs)
+
+set(SYCL_ROOT "")
+if(DEFINED ENV{SYCL_ROOT})
+  set(SYCL_ROOT $ENV{SYCL_ROOT})
+elseif(DEFINED ENV{CMPLR_ROOT})
+  set(SYCL_ROOT $ENV{CMPLR_ROOT})
+endif()
+
+string(COMPARE EQUAL "${SYCL_ROOT}" "" nosyclfound)
+if(nosyclfound)
+  set(SYCL_FOUND False)
+  set(SYCL_REASON_FAILURE "SYCL library not set!!")
```

Comment:
How about "SYCL_ROOT environment variable is not defined"

Reply:
If `SYCL_ROOT` is undefined, `PYTORCH_FOUND_XPU` in xpu.camke will be set to `False`, and update `USE_XPU` to `False` in Dependencies.cmake